### DISCUSSION TITLE
2023-12-14 Users Community Call

### DISCUSSION THREAD

**Discussion by ahnaf-tahmid-chowdhury** [svalinn/DAGMC#930] [Community Calls]
Attending:

ANL: @pshriwise 
First Light Fusion: @shimwell
Copenhagen Atomics: @ebknudsen
TÜV SÜD Nuclear Technologies: @micksalt
BaMSciConsulting: @bam241
MIST: @ahnaf-tahmid-chowdhury

**Comment by pshriwise** [comment]
Discussed:
  - updates to [cad_to_openmc](https://github.com/openmsr/CAD_to_OpenMC)
  - FluDAG interface and testing for CERN's distribution of Fluka (#677)
  - MOAB version issues. Use <= 5.3.1 for now
  - Discussion on [PyDAGMC](https://github.com/svalinn/pydagmc) roadmap -- in short, it's very open-ended

**Comment by shimwell** [comment]
We also discussed adding cad-to-openmc testing to the growing [model benchmark zoo](https://github.com/fusion-energy/model_benchmark_zoo).

@ebknudsen has just added a test for one of the shapes
